package.json

```diff
@@ -13,7 +13,7 @@
     "dev:dynamo": "node bootstrap/local-dynamo.js",
     "test": "jest --watch --runInBand",
     "test:ci": "jest --ci --runInBand",
-    "compile": "babel src -d lib && babel config -d lib/config"
+    "compile": "babel src -d build/src && babel config -d build/config"
```

Comment:
I fixed a typo here: `-d lib` should have been `-d lib/src`, but also renamed `lib` to `build` to be clearer.